localization/languages/ko.json

```diff
@@ -97,53 +97,53 @@
-    "sslErrorTitle": "누리집을 사용할 수 없습니다.",
-    "sslErrorMessage": "안전한 연결을 할 수 없습니다.",
-    "dnsErrorTitle": "누리집을 찾을 수 없습니다.",
+    "sslErrorTitle": "사이트를 사용할 수 없습니다.",
+    "sslErrorMessage": "SSL 인증이 실패했습니다.",
+    "dnsErrorTitle": "사이트를 찾을 수 없습니다.",
     "dnsErrorMessage": "DNS 오류가 발생했습니다.",
-    "offlineErrorTitle": "통신망에 연결되지 않은 상태입니다.",
-    "offlineErrorMessage": "통신망에 재연결 후에, 다시 시도해주세요.",
-    "genericConnectionFail": "누리집에 연결할 수 없습니다.",
-    "sslTimeErrorMessage": "안전한 연결을 할 수 없습니다. 사용 장비의 시간 설정을 확인해주세요.",
-    "addressInvalidTitle": "유효하지 않은 주소입니다.",
+    "offlineErrorTitle": "인터넷에 연결되지 않은 상태입니다.",
+    "offlineErrorMessage": "인터넷이 연결되면 다시 시도해주세요.",
+    "genericConnectionFail": "사이트에 접속할 수 없습니다.",
+    "sslTimeErrorMessage": "안전한 연결을 할 수 없습니다. 장치의 시간이 바르게 설정되어 있는지 확인해주세요.",
+    "addressInvalidTitle": "유효하지 않은 주소",
     "genericError": "오류 발생",
     /* pages/phishing/index.html */
-    "phishingErrorTitle": "유해한 누리집입니다.",
-    "phishingErrorMessage": "이 누리집에서는 암호 또는 은행 정보와 같은 개인 정보가 도용될 수 있습니다.",
-    "phishingErrorVisitAnyway": "(그럼에도 불구하고) 들어가기",
+    "phishingErrorTitle": "위험한 사이트입니다.",
+    "phishingErrorMessage": "이 사이트에서는 암호 또는 은행 정보와 같은 개인 정보가 도용될 수 있습니다.",
+    "phishingErrorVisitAnyway": "(위험을 감수하고) 들어가기",
     "phishingErrorLeave": "나가기",
     /* multiple instances alert */
     "multipleInstancesErrorMessage": "오류가 발생했습니다. 다른 창을 종료 후 재실행해주세요.",
     /* pages/sessionRestoreError/index.html */
     "sessionRestoreErrorTitle": "오류 발생",
-    "sessionRestoreErrorExplanation": "저장된 작업 자료를 정상적으로 복원할 수 없습니다.",
-    "sessionRestoreErrorBackupInfo": "복원용 작업 자료 저장위치: %l.", //%l will be replaced with a path to a file
+    "sessionRestoreErrorExplanation": "저장된 파일를 정상적으로 복원할 수 없습니다.",
+    "sessionRestoreErrorBackupInfo": "복원용 파일 저장위치: %l.", //%l will be replaced with a path to a file
     "sessionRestoreErrorLinkInfo": {
       "unsafeHTML": "오류가 계속 발생하면 <a href=\"https://github.com/minbrowser/min\" target=\"_blank\">문의</a> 바랍니다."
     },
     /* pages/settings/index.html */
-    "settingsPreferencesHeading": "환경설정",
+    "settingsPreferencesHeading": "설정",
     "settingsRestartRequired": "변경 사항을 적용하려면 다시 시작해야 합니다.",
-    "settingsPrivacyHeading": "내용물 차단",
+    "settingsPrivacyHeading": "추적기 차단",
     "settingsContentBlockingLevel0": "모든 광고와 추적 허용",
     "settingsContentBlockingLevel1": "타사 광고와 추적 차단",
     "settingsContentBlockingLevel2": "모든 광고와 추적 차단",
-    "settingsContentBlockingExceptions": "다음의 누리집에서 광고 허용",
+    "settingsContentBlockingExceptions": "다음의 사이트에서 추적 허용",
     "settingsBlockScriptsToggle": "명령(스크립트) 차단",
-    "settingsBlockImagesToggle": "그림 차단",
+    "settingsBlockImagesToggle": "이미지 차단",
     "settingsBlockedRequestCount": {
-      "unsafeHTML": "지금까지 민(Min)은 <strong></strong> 광고 및 추적기를 차단했습니다."
+      "unsafeHTML": "지금까지 차단된 광고 및 추적기: <strong></strong>"
     },
-    "settingsCustomBangs": "사용자 정의 연결 명령",
-    "settingsCustomBangsAdd": "사용자 정의 연결 명령 추가",
+    "settingsCustomBangs": "Custom Bang Search",
```

Comment:
If it's not possible to translate this, can you leave it as "custom bangs" as in the English version?

Reply:
OK, I will edit it.